packages/wp-story-editor/src/api/index.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export * from './authors';
+export * from './getStory';
+export * from './media';
+export * from './metaboxes';
+export * from './metadata';
+export * from './pageTemplate';
+export * from './saveStory';
+export * from './statusCheck';
```

Comment:
Nit: this exposes `statusCheck` callbacks to `story-editor`, even though it's a `wp-story-editor` specific util.

Reply:
Same for the `metaboxes` export above

Reply:
Noted, and added `@todo`, will remove them as I work on moving meta boxes, StatusCheck, and PostLock components.

Reply:
Since those still exist in the `story-editor`, they will be required in `APIProivder` for now.